packages/create-app/templates/default-app/README.md

```diff
@@ -1,3 +1,10 @@
 # [Backstage](https://backstage.io)
 
 This is your newly scaffolded Backstage App, Good Luck!
+
+To start the app, run:
+
+```sh
+$ yarn
+$ yarn dev
```

Comment:
We avoid prefixing with `$` nowadays, so let's skip it here. Same as above as well with the `yarn install`

Reply:
Good to know, thanks! Also resolved in a54fdab36e32aa796963d901706a13e477fb6eb9